### PR TITLE
VIDEO-11239 Add Krisp Noise Cancellation

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-11239](https://issues.corp.twilio.com/browse/VIDEO-11239)

### Description

This PR adds Krisp to the Ahoy app. If Krisp is installed, users can enable and disable noise cancellation by clicking the switch button on the Device Selection Screen before joining the room, and any time during the call by accessing Audio and Video settings.

krisp

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [X] Included screenshot as PR comment (if needed)
* [X] Ready for review

### Before merge
* [x] Got one or more +1s
* [x] Re-tested if necessary

### CODE REVIEW

**Review by PikaJoyce — DISMISSED** [review]
Thanks @olipyskoty! Looks good!

**Review by PikaJoyce — DISMISSED** [review]

File: src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx [hunk 1]
```diff
@@ -39,7 +39,12 @@ describe('the useLocalTracks hook', () => {
       });
 
       expect(Video.createLocalTracks).toHaveBeenCalledWith({
-        audio: true,
+        audio: {
+          noiseCancellationOptions: {
+            sdkAssetsPath: '/noisecancellation',
+            vendor: 'krisp',
+          },
```
Comment (PikaJoyce) [inline src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx:46]: It seems like we re-use this multiple times in this file. Can we store this as a variable once and reuse that?

**Review by PikaJoyce — COMMENTED** [review]
Awesome work @manjeshbhargav! 

Left one small comment but otherwise it LGTM!

File: src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx (hunk 1 above)
Comment (manjeshbhargav) [reply src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx:46]: @PikaJoyce I'll do that in a separate PR. Now, I'm focused on getting this out.

**Review by PikaJoyce — APPROVED** [review]

**Comment by fschwahn** [comment]
Is there a possibility to disable this feature? I'm getting errors in the Firefox console, and am unable to join a room with version 0.9.0.

**Pull request merged**